ci-operator/config/openshift/coredns-ocp-dnsnameresolver/openshift-coredns-ocp-dnsnameresolver-main.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
```

Comment:
Do we need the `base_images`? There is no `images` section in this CI job. That is, no images is built by the job. Normally we use `base_images` in `images` section to make some substitutions in `Dockerfile`'s `FROM` instruction.